ApprovalTests/reporters/LinuxReporters.h

```diff
@@ -33,6 +75,10 @@ namespace ApprovalTests
                 : FirstWorkingReporter({
                       // begin-snippet: linux_diff_reporters
                       new MeldReporter(),
+                      new SublimeMergeSnapReporter(),
+                      new SublimeMergeFlatpakReporter(),
+                      new SublimeMergeReporter(),
+                      new SublimeMergeTarballReporter(),
```

Comment:
I wondered if you had any comments or thoughts about creating a `SublimeMergeReporter` class for Linux that is a `FirstWorkingReporter` of the 4 classes listed here - so that users could use that class on all 3 platforms... In Windows and/or Mac there are one or two similar classes that use `FirstWorkingReporter` to group together related diff tools - from memory, like BeyondCompare...

Again, I'm happy to create this after merge, but wondered about your opinion first...

(I know it's more boilerplate code... ☹️  but perhaps more a bit more convenient and consistent for users?)

Reply:
That sounds like a great idea to me, I just wasn't aware of the `FirstWorkingReporter` class. I'll make the change.